### ISSUE TITLE
how to set unlable data from training dataset?

### ISSUE THREAD

**Issue by Chriszhangmw** [GT-SALT/MixText#11]
Hi, I am trying to implement your code in Chinese datasets, for the function get_data, your code split label and un-lable samples by: train_unlabeled_idxs.extend(idxs[500: 500 + 5000]), but idxs comes from idxs = np.where(labels == i)[0], so I think all the idxs are lable data, why you choose unlable from idxs?  we will appreciate if you can upload some sample datasets on your experiment. thanks

**Comment by jiaaoc** [comment]
Hi, the idxs = np.where(labels == i)[0] is to select data from class i. In this work, we randomly sampled data from the original training set to form the unlabeled dataset. And during training, model can not get access to the corresponding labels for these idxs. The example data (Yahoo!Answer) have been uploaded here (https://drive.google.com/file/d/1IoX9dp_RUHwIVA2_kJgHCWBOLHsV9V7A/view?usp=sharing)
